src/main/java/org/killbill/billing/plugin/adyen/client/payment/service/AdyenCheckoutApiClient.java

```diff
@@ -23,18 +24,23 @@
     private static final Logger logger = LoggerFactory.getLogger(AdyenCheckoutApiClient.class);
 
 
-    public AdyenCheckoutApiClient(final AdyenConfigProperties adyenConfigProperties) {
+    public AdyenCheckoutApiClient(final AdyenConfigProperties adyenConfigProperties, final String countryCode) {
         // initialize the REST client here
         Environment environment = Environment.TEST; //default Adyen environmnet
```

Comment:
Can you add a java doc related to the Adyen Test and Prod client.
It will create confusion for developers to use same Test client for Dev, UAT and Staging env.

Reply:
+1